### PR TITLE
Documentation for the export feature

### LINKED ISSUE #839: Document export functionality
@kpavel can you please add a doc on export functionality under https://github.com/apache/incubator-openwhisk-wskdeploy/tree/master/docs

Please describe the details such as:

1. What does it do? How its useful?
1. `wskdeploy` command line for export functionality
2. Sample manifest files exported using this functionality.

### PR DESCRIPTION
**Pull request by davidbreitgand**
Added documentation for the export feature.
1. README.md: brief mention of the feature and the reference to
2. docs/export.md -- describes the usage 

Closes #839

### CODE REVIEW

File: docs/export.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+The manifest(s) resulting from executing `wskdeploy export` can be later used for deploying at a different `OpenWhisk` instance. The code of actions, which are defined in the packages of the exported project will be saved into folders with the names being the names of the package, the actions belong to.
+
+## Use Cases
+
+### Copy `OpenWhisk` Assets
+
+One common scenario, in which the export feature is useful, is populating a newly installed `OpenWhisk` instance with assets from 
+another `OpenWhisk` instance. One might consider a scenario, in which an `OpenWhisk` instance is installed on premises with another `OpenWhisk` instance residing in the cloud. Consider, for example using one `OpenWhisk` instance on premises and another one in the cloud (e.g., the second `OpenWhisk` instance can be [IBM Cloud Functions](https://console.bluemix.net/openwhisk/)). A fairly common scenario is that a developer
```
[the remark this answers is not in the document]
Comment (mrutkows) [reply docs/export.md]: @davidbreitgand @pritidesai our docs should be devoid of any implementation reference where possible. Is there a way to generalize the use case?  provider A to provider B

[the remark this answers is not in the document]
Comment (davidbreitgand) [reply docs/export.md]: Sure, I can drop reference to IBM Cloud Functions.

**Comment by davidbreitgand** [comment]
Yes, will do this week. I'm on a hectic schedule for the last couple weeks. Sorry about delay.

**Comment by davidbreitgand** [comment]
Tx, Priti. I'll open a new PR to fix it this week.

**Pull request merged**